from celery.contrib import rdb; rdb.set_trace()